x-pack/test/reporting_api_integration/reporting_and_security/usage/api_counters.ts

```diff
@@ -29,11 +29,14 @@ export default function ({ getService }: FtrProviderContext) {
       await reportingAPI.teardownEcommerce();
     });
 
-    it('configuration settings of the tests_server', async () => {
-      const usage = await usageAPI.getUsageStats();
-      expect(usage.kibana_config_usage.xpack_reporting_capture_max_attempts).to.be(1);
-      expect(usage.kibana_config_usage.xpack_reporting_csv_max_size_bytes).to.be(6000);
-      expect(usage.kibana_config_usage.xpack_reporting_roles_enabled).to.be(false);
+    describe('server', function () {
+      this.tags('skipCloud');
+      it('configuration settings of the tests_server', async () => {
+        const usage = await usageAPI.getUsageStats();
+        expect(usage.kibana_config_usage.xpack_reporting_capture_max_attempts).to.be(1);
+        expect(usage.kibana_config_usage.xpack_reporting_csv_max_size_bytes).to.be(6000);
+        expect(usage.kibana_config_usage.xpack_reporting_roles_enabled).to.be(false);
+      });
```

Comment:
@tsullivan wdyt?

Reply:
@vadimkibana @liza-mae looks like this test can only be used to verify on-prem capabilities. Based on [this issue](https://github.com/elastic/kibana/issues/139141), it looks like there isn't a way to set up a cloud instance to meet the configuration for this test. Skipping makes sense to me.

Reply:
Thanks!